infra/main.bicep

```diff
@@ -18,7 +18,7 @@ param searchServiceResourceGroupName string = ''
 param searchServiceResourceGroupLocation string = location
 
 param searchServiceSkuName string = 'standard'
-param searchIndexName string = 'gptkbindex'
+param searchIndexName string = ''
```

Comment:
I think actually the right approach here is what we did for the deployment names - don't set this to any value, like:

```
param gptDeploymentName string // Set in main.parameters.json
```

Then in main.parameters.json, do:

```
   "searchIndexName": {
      "value": "${AZURE_SEARCH_INDEX=gptkbindex}"
    },
```

I don't recall why right now, but I think this works better in more situations.

Reply:
Updated to the requested pattern